### DISCUSSION TITLE
Support for Steady State MT3DMS Mode

### DISCUSSION THREAD

**Discussion by AKoehler95** [modflowpy/flopy#1508] [Q&A] [answered]
Hi everyone,

I was wondering if there is support for the steady state mode for transport modeling as mentioned in the MT3DMS manual for version 5.3, chapter 10 (https://hydro.geo.ua.edu/mt3d/mt3dms_v5_supplemental.pdf) in flopy.
In the documentation I could not find a "steady state" argument for the BTN module and was wondering if this can be done or if the period length must be adjusted to some sufficiently long time.
Thanks in advance!

Cheers,
Anton

**Comment by jlarsen-usgs** [comment, marked as answer]
I checked the source code today and the ssflag argument does exist for the BTN module, but was not added to the documentation. Here's an example of how to use it:

```python
ssflag = ["ssflag"] + ["" for _ in range(kper - 1)]
btn = Mt3dBtn(..., ssflag=ssflag)
 # or if your btn file already exists you can use
btn.ssflag = ssflag
```

**Reply by AKoehler95** [reply]
Thank you, works now!
